src/Console/Command/PostExporter.php

```diff
@@ -32,53 +32,54 @@
 class PostExporter extends Command
 {
 
-	use UserContext;
 	use FormatterTrait;
 
 	private $data;
 	private $postExportRepository;
 	private $exportJobRepository;
 	private $formAttributeRepository;
 
-    /**
-     * The console command name.
-     *
-     * @var string
-     */
-    protected $name = 'export';
-
-    /**
-     * The console command signature.
-     *
-     * @var string
-     */
-    protected $signature = 'export {--limit=100} {--offset=0}';
-
-    /**
-     * The console command description.
-     *
-     * @var string
-     */
-    protected $description = 'Export posts';
+	/**
+	 * The console command name.
+	 *
+	 * @var string
+	 */
+	protected $name = 'export';
+
+	/**
+	 * The console command signature.
+	 *
+	 * @var string
+	 */
+	protected $signature = 'export {--limit=100} {--offset=0} {--job} {--include-header=1}';
```

Comment:
Are these meant to be defaults?

Reply:
Yup

Reply:
Lumen command definitions are much easier